hw1/01-selectors.md

```diff
@@ -1,7 +1,37 @@
 ## CSS Selectors - CSS Diner Game
 
+Travis Block
+Thursday, September 28th 2023
+
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple
+5. #fancy pickle
+6. .small
+7. orange.small
+8. bento orange.small
+9. plate,bento
+10. *
+11. plate *
+12. plate + apple
+13. bento ~ pickle
+14. plate > apple
+15. plate :first-child or plate *:first-child
+16. plate :only-child or plate *:only-child
+17. .small:last-child
+18. plate:nth-child(3)
+19. bento:nth-last-child(3)
+20. apple:first-of-type
+21. plate:nth-of-type(even)
+22. plate:nth-of-type(2n+3)
+23. plate apple:only-of-type
+24. orange:last-of-type, apple:last-of-type
+25. bento:empty
+26. apple:not(.small)
+27. *[for]
```

Comment:
I believe the star selector here is unnecessary. Removing it and leaving just `[for]` should accomplish the same result.